src/simstation/StatsCommand.java

```diff
@@ -2,16 +2,30 @@
 
 import mvc.Command;
 import mvc.Model;
+import javax.swing.*;
 
 public class StatsCommand extends Command {
-    private Model model;
```

Comment:
Again I think this is not the way we want to save the model. Look into the command class. Model is declared and we don't need a variable to save it.